Translation/en/RedirectedResources/assets/abdata/h/list/01/personality_voice_c25/translation.txt

```diff
@@ -11,7 +11,7 @@
 いっぱい気持ちよくしてあげる〜=I'll make you feel amazing~
 ……始めよう……ね？=...Let's start... okay?
 素敵な彼女さんといいことしよ〜=Come on, let's do some fun stuff with your lovely girlfriend~
-［P］の好きにしていいよ〜=You can do it however you like, ［P］~
+［P］の好きにしていいよ〜=You can do it however you like, [P]~
```

Comment:
These appear to be the wrong characters. The original strings use:
`［` and `］`, which is what they had before, the new ones are using `[` and `]`. Does the game handle both, or are these not going to get subbed?

Reply:
I've always used normal [ ] versions in both KK and KKS and didn't hear of any issues. Worst case, this will be a simple replace all once everything is merged.

Reply:
Just checked and the game itself usually uses the ASCII brackets, it seems like this is just an oddity that only appears in c10 and c25.